### DISCUSSION TITLE
Using .env files in URLs

### DISCUSSION THREAD

**Discussion by fnuttens** [Julien-cpsn/ATAC#73] [Q&A] [answered]
Since there’s currently no doc for this feature in the wiki I have trouble finding the good way to create URL templates.

Here’s what I tried: I added KEY=value fields in a new .env file that ATAC is `Checking` at startup.
In the URL I figured I’d use the following template style:

```
https://{{KEY}}/path
```

But it doesn’t work :(

**Comment by Julien-cpsn** [comment, marked as answer]
Hello!

What is file name of your environment? If it is only ".env" then it will probably not be used. You have to name it something like ".env.my_env" to get it parsed. If it gets parsed correctly, a gray box will appear en top left corner when using the app.

I am really sorry for the lack of documentation, I am trying to release a strong v1.0.0 in order to write the whole documentation 😄

I would say that help is needed on that side haha

Have a great day

**Reply by fnuttens** [reply]
Hey! It works now, thank you 🙂 
I was initially trying with a `X.env` file.

No need to apologize, the lack of doc is totally understandable given the youth of this (promising) project 😉 
Can’t wait to kiss Insomnia goodbye 😄
